pinot-common/src/thrift/request.thrift

```diff
@@ -125,6 +125,7 @@ struct BrokerRequest {
  10: optional string bucketHashKey;
  11: optional bool enableTrace;
  12: optional string responseFormat;
+ 13: optional list<string> routingOptions;
```

Comment:
why not map and rename it to debugOptions. That will allow us to add more options without changing the thrift protocol. Note none of this should be exposed to users, this is for dev only for debugging.